python/cuml/tests/test_linear_svm.py

```diff
@@ -286,8 +385,9 @@ def run_sklearn():
         stress_param((100000, 1000)),
     ],
 )
-def test_classification_1(datatype, penalty, loss, dims):
-    run_classification(datatype, penalty, loss, dims, 2)
+@pytest.mark.parametrize("class_weight", ["balanced", {0: 0.5, 1: 1.5}])
```

Comment:
Depending on how long does `test_classification_1` takes, one could consider not adding a tensor product with `class_weight` param, but have a separate test where only the class weight changes, and rest is fixed.